### ISSUE TITLE
How efficient for large data set, say - 2000 data

### ISSUE THREAD

**Issue by kirtishjain9** [kkemple/react-native-sideswipe#64]
Thanks a lot for providing this excellent library but I wonder about the efficiency of this library for large data set say 2000 data in my data array.

How fast these 2000 data will be loaded and how smooth the scrolling between the individual cards (dynamic card).

Kindly suggest me.

Thanks in advance.

**Comment by zolero** [comment]
Just try it?

**Comment by kirtishjain9** [comment]
Yes, I had given a try but data loading is taking enough time to show.

How could I make it efficient as I have to load around 2000 data cards.

Thanks in advance.

**Comment by pedro-lb** [comment]
The ideal solution would be to not load that much data. In any scenario, doing this is a performance harm.

I don't know where you are loading the data from - but perhaps you could add `limit` and `skip` options in order to paginate (e.g. load the first 10 cards, and when the user reaches the end you load 10 more, and so on).
